### PR TITLE
[#2] Added tests to cryptocurrencies network information classes

### PR DESCRIPTION
**Pull request by szerepak**

### CODE REVIEW

File: tests/test_network.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    assert isinstance(cls.seeds, tuple) is True
+    assert isinstance(cls.port, int) is True
+
+
+@mark.parametrize('seed', seeds)
+def test_seeds_valid_dns_address(seed):
+    assert domain(seed) is True
+
+
+@mark.tox_not_run
```
Comment (trojkat) [inline tests/test_network.py]: `exclude_from_ci` as we shouldn't mention CI tool directly.

File: setup.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
     long_description=long_description,
     install_requires=[
         'bitcoinlib==0.3.29',
+        'validators==0.12.0',
```
Comment (trojkat) [inline setup.py]: Move this one to `extras_require`.

**Review by trojkat — APPROVED** [review]
A piece of nice code! 👍

**Review by pafff — APPROVED** [review]
Great! :+1:

**Pull request merged**